main.html.in

```diff
@@ -14,6 +14,9 @@
   <!-- Load config scripts separately because compiled-method-cache.js
      - depends on them. -->
   <script type="text/javascript" src="config/default.js" defer></script>
+  <!-- #if CONFIG -->
+  <script type="text/javascript" src="@CONFIG@" defer></script>
+  <!-- #endif -->
   <script type="text/javascript" src="config/midlet.js" defer></script>
```

Comment:
> Can we remove config/midlet.js and merge it with the new @CONFIG@ file?

Yes, good idea, I've done so in a13b030, except that I merge it with default.js, which seems like a stabler target and also more apropo, since these are default values.